client-app/components/organisms/table/_internal/table-status-badge/table-status-badge.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <div class="w-20 py-1 text-center rounded-sm" :class="badgeClass">{{ status }}</div>
+</template>
+<script setup lang="ts">
+import { computed } from "vue";
+
+const props = defineProps({
+  status: {
+    type: String,
+    default: null,
+  },
+});
+
+const newStatusClass = "bg-green-500 text-white";
+const processingStatusClass = "border border-green-500 text-green-500";
+const pandingStatusCalss = "border border-blue-400 text-blue-400";
+
+const badgeClass = computed(() => {
+  return {
+    [newStatusClass]: props.status === "New",
```

Comment:
This component should be in the shared folder. Try not to hardcode sensitive display logic into logicless components.
Maybe it's better to create extendable cell slot and pass this component into it.